src/main/java/com/adyen/util/Util.java

```diff
@@ -159,4 +160,10 @@ public static String calculateSessionValidity() {
 
         return fmt.format(sessionDate);
     }
+
+    public static String getPaginationFragment() {
```

Comment:
Is the format actually something generic which other APIs are using as well, or specific to the `Management API`.

Reply:
I haven't noticed any usage of pagination in other parts of the library. Perhaps it can be moved to something more centralized later on.